.github/workflows/daily-test-build-numpy.yml

```diff
@@ -69,7 +69,6 @@ jobs:
 
       - name: Test without pandas
         run: python -m pip uninstall -y pandas
-        if: ${{ matrix.uninstall_pandas }}
```

Comment:
I don't think this line should be deleted. We want to run the unit tests with and without pandas.

Reply:
This file is not parameterized on that AFAICT...